extensions/fairifier-rdf-extension/dtls/fairifier/RdfSkeletonService.java

```diff
@@ -13,8 +13,8 @@
  */
 
 interface RdfSkeletonService {
-    public List<String[]> listModels(String fileType) throws IOException;
-    public List<String[]> listModels()  throws IOException;
+    public List<RdfSkeletonTransformer> listModels(String fileType) throws IOException;
+    public List<RdfSkeletonTransformer> listModels()  throws IOException;
```

Comment:
Could you explain the rationale behind returning the transformer instead of a domain object?